src/tools/demarchesSimplifiees/affichageDifferenceQuota.ts

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env node
+/* eslint-disable no-await-in-loop */
+
+// Lancement de ce script : ts-node src/tools/demarchesSimplifiees/affichageDifferenceQuota.ts
+
+import { GraphQLClient } from 'graphql-request';
+import execute from '../utils';
+import service from '../../helpers/services';
+import { IDossierDS } from '../../ts/interfaces/global.interfaces';
+import { queryGetDemarcheDemarcheSimplifiee } from '../../services/structures/repository/demarchesSimplifiees.repository';
+
+const requestGraphQLForGetDemarcheDS = (
+  graphQLClient: GraphQLClient,
+  demarcheNumber: string,
+  cursor: string,
+): Promise<any> =>
+  graphQLClient
+    .request(queryGetDemarcheDemarcheSimplifiee(), {
+      demarcheNumber: Number(demarcheNumber),
+      after: cursor,
+    })
+    .catch(() => {
+      return new Error("La démarche n'existe pas");
+    });
+
+execute(
+  __filename,
+  async ({ app, logger, exit, graphQLClient, demarcheSimplifiee }) => {
+    const arrayCursor: string[] = ['', '', ''];
+    const arrayHasNextPage: boolean[] = [true, true, true];
+    let dossiersStructurePubliqueBrut = [];
+    let dossiersEntrepriseEssBrut = [];
+    let dossiersAssociationsBrut = [];
+
+    do {
+      if (arrayHasNextPage[0] === true) {
```

Comment:
Est-ce qu'on peut simplifier comme ça ?
```suggestion
      if (arrayHasNextPage[0]) {
```